source/jormungandr/jormungandr/street_network/parking/augeas.py

```diff
@@ -0,0 +1,85 @@
+#  Copyright (c) 2001-2019, Kisio Digital and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Kisio Digital (http://www.kisiodigital.com/).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+from __future__ import absolute_import, print_function, unicode_literals, division
+import requests, pybreaker, logging
+from jormungandr.street_network.parking.abstract_parking_module import AbstractParkingModule
+
+
+class Augeas(AbstractParkingModule):
+    def __init__(self, service_url, max_park_duration=1200):
```

Comment:
It's quite confusing and obfuscated when duration types are neither explicit nor documented. 
I guess your duration here is in second because that's an assumption we've made in the codebase. But to withdraw the ambiguity, I would rather see `max_park_duration=time(0, 0, 1200)` or even better `time(0, 20)`.